test/utils/fit-bounds.spec.js

```diff
@@ -1,20 +1,72 @@
 import test from 'tape-catch';
 import {fitBounds} from 'react-map-gl';
 
+/* fitBounds from Mapbox for comparison */
+import Transform from 'mapbox-gl/js/geo/transform';
```

Comment:
Possible, but since this is only used for testing, the module itself won't be affected.